code/modules/tips/tips.dm

```diff
@@ -0,0 +1,200 @@
+	weight = 1
+
+/datum/tip_of_the_day/teaching
+	title = "Teach"
+	desc = "Teaching your fellow spaceman is a time honored tradition. Help the newbies out, pass on the torch."
+	category = "Roleplay"
+	weight = 3
+
+/datum/tip_of_the_day/forever
+	title = "There is never an end to ss13"
+	desc = "Don't forget. You're here forever."
+	category = "Roleplay"
+	weight = 1
+
+// -- General
+
+/datum/tip_of_the_day/ground_up
+	title = "Grinding items"
+	desc = "A lot of things can be ground up to make chemicals and materials. Try and grind up a Danitos package, if you dare!"
+	category = "General"
+	weight = 3
+
+/datum/tip_of_the_day/alt_click
+	title = "Containers"
+	desc = "You can alt-click containers to show their contents and put items in them without picking them up."
+	category = "General"
+	weight = 3
+
+/datum/tip_of_the_day/hotkey_mode
+	title = "Hotkey mode"
+	desc = "You can press TAB to enable hotkey mode and use WASD movement, T for talk, and other keyboard shortcuts. Consult the wiki for a full list."
+	category = "General"
+	weight = 3
+
+/datum/tip_of_the_day/pda_messages
+	title = "PDAs"
+	desc = "Your PDA has a lot of utility. Buy some programs for it, or send everyone in a department a message if they're not listening to the radio! Some can even be health scanners!"
+	category = "General"
+	weight = 3
+
+// -- Command
+
+/datum/tip_of_the_day/chain_of_command
+	title = "Chain of command"
+	desc = "When there is no captain at roundstart, other heads of staff can become acting captain and secure important items such as the captain's spare and do job changes."
+	category = "Silicon laws"
+	weight = 3
+
+// -- Silicon laws
+
+/datum/tip_of_the_day/emotional_harm
+	title = "Emotional harm"
+	desc = "On the Asimov lawset, emotional harm is not considered human harm."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/tip_of_the_day/self_harm
+	title = "Self harm"
+	desc = "On the Asimov lawset, self-harm is not considered human harm."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/tip_of_the_day/burgs
+	title = "Robot names"
+	desc = "As a cyborg, minor misspelling of your name are NOT a good reason to ignore orders."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/tip_of_the_day/law_orders
+	title = "Self harm"
+	desc = "When they blatantly conflict, silicon laws are to be interpreted in descending order - higher laws have higher priority - unless otherwise specified. When obeying a higher law, you must avoid violating lower laws if possible."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/tip_of_the_day/burgs
+	title = "Name missepelling"
+	desc = "As a cyborg, minor misspelling of your name are NOT a good reason to ignore orders."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/tip_of_the_day/slaved_borg
+	title = "Cyborgs are slaved to their AI"
+	desc = "As a cyborg slaved to an AI, you are expected (as far as reasonably possible) to obey your AI when it does not come into conflicts with your laws."
```

Comment:
i thought you were to unconditionally obey your master AI?
in case of conflicts ahelp